tasks/ss-main.yml

```diff
@@ -207,9 +197,56 @@
 
 
 ###########################################################
-#   5- Database config - moved to separate files
+#   5- SS worker
+###########################################################
+
+- name: "Check if django-celery is in the pip requirements"
+  command: "grep '^django-celery' base.txt"
+  args:
+    chdir: "{{ archivematica_src_dir }}/archivematica-storage-service/requirements"
+  register: "grep_celery"
+  tags: "amsrc-ss-worker"
+
+- set_fact:
+    ss_worker: true
+  when: "grep_celery.stdout!=''"
```

Comment:
Perhaps more accurate to rely on the exit code of grep instead of stdout? Should be zero if there was at least one match.